CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
-## v1.0.1
-### 2024-04-05
+# Changelog
+
+## [v1.1.0] - 2024-08-29
```

Comment:
I like this update to the change log format, but you'll need to make a quick change to [the script](https://github.com/nasa/harmony-swath-projector/blob/main/bin/extract-release-notes.sh) that extracts information from here to make the release notes.

Reply:
I never remember this.  Thanks [ed0ff0e](https://github.com/nasa/harmony-swath-projector/pull/14/commits/ed0ff0e7931b8c636826c8453ed8a27a8f8eb104)